include/nccl-headers/nvidia/tuner.h

```diff
@@ -1,6 +1,7 @@
 /*************************************************************************
  * Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
  * Copyright (c) 2023, Meta Platforms, Inc. and affiliates.
+ * Copyright (c) 2024, Amazon.com, Inc. or its affiliates. All rights reserved.
```

Comment:
Nack. This should be a direct lift from the NVIDIA header right?

Reply:
yes, I changed this in an earlier version, but I agree now it should not be there